test/IntegrationTestCommon/Factories/WebApplicationFactoryExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.Primitives;
+
+namespace Bit.IntegrationTestCommon.Factories
+{
+    public static class WebApplicationFactoryExtensions
+    {
+        public static async Task<HttpContext> SendAsync(this TestServer server,
```

Comment:
I know you don't have extension methods for `Put` or `Delete`, but I think you can still just mark this private if you're gonna go with the `PostAsync`/`GetAsync` route